controllers/dopplersecret_controller_secrets.go

```diff
@@ -181,7 +191,8 @@ func (r *DopplerSecretReconciler) CreateManagedSecret(ctx context.Context, doppl
 
 // UpdateManagedSecret updates a managed Kubernetes secret
 func (r *DopplerSecretReconciler) UpdateManagedSecret(ctx context.Context, secret corev1.Secret, dopplerSecret secretsv1alpha1.DopplerSecret, secretsResult models.SecretsResult) error {
-	secretData, dataErr := GetKubeSecretData(secretsResult, dopplerSecret.Spec.Processors)
+	includeSecretsByDefault := dopplerSecret.Spec.ManagedSecretRef.Type == string(corev1.SecretTypeOpaque)
```

Comment:
nit: This feels like it is not idiomatic. I think the more common paradigm is relying on a boolean's zero value, and then explicity setting the value if the condition passes. It is more verbose, but simpler to read.
```go
var includeSecretsByDefault bool
if  dopplerSecret.Spec.ManagedSecretRef.Type == string(corev1.SecretTypeOpaque) { 
  includeSecretsByDefault = true
}
```